package.json

```diff
@@ -89,7 +89,7 @@
     "@discordjs/opus": "^0.8.0",
     "@discordjs/rest": "1.0.1",
     "@discordjs/voice": "0.17.0",
-    "@distube/ytdl-core": "^4.15.9",
+    "@distube/ytdl-core": "^4.16.2",
```

Comment:
If you gander at https://github.com/museofficial/muse/pull/1195 and https://github.com/museofficial/muse/pull/1180 

What is different exactly? It's been an issue for a few months now, and has a few of us... well stumped.

Reply:
Good question.
In my PR, I only upgraded the version of ytdl-core to get music downloads working again.
https://github.com/museofficial/muse/pull/1180 do the same, but I needed to update to version 4.16 because there were changes in the page's regex.

Reply:
I update the node version to 20, to prevent some errors too